infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/SessionManager.java

```diff
@@ -266,13 +257,37 @@ private void createSessions() {
                 incomingSessions.size(), sessions);
     }
 
+
     private void logNewlyAddedSessionInfo(Set<LogReplicationSession> newlyAddedSessions) {
         log.info("========= HashCode -> Session mapping for newly added session: =========");
         for (LogReplicationSession session : newlyAddedSessions) {
             log.info("HashCode: {}, Session: {}", session.hashCode(), session);
         }
     }
 
+    /**
+     * Build a session and add to incoming/outgoing session appropriately.
+     */
+    private void sessionBuilder(String sourceClusterId, String sinkClusterId,
```

Comment:
nit: since this is a method and not an object, can we call it something like `constructSession` or something similar?